src/components/Collection/Collection.tsx

```diff
@@ -9,12 +9,19 @@ type PropTypes = {
   header?: React.ReactNode
   children: React.ReactNode | React.ReactNode[]
   footer?: React.ReactNode
+  className?: string
 }
 
-const Collection = ({ title = '', children, header, footer }: PropTypes) => {
+const Collection = ({
+  title = '',
+  children,
+  header,
+  footer,
+  className = '',
+}: PropTypes) => {
   return (
     <Widget
-      className={styles.collection}
+      className={className}
```

Comment:
Question: I'm not sure but do we need to combine these classes? Or does this make `src/components/Collection/Collection.module.scss` unnecessary?